From 919465fbdca1021735b7e80ad827a84809ada29a Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Tue, 12 Jul 2022 02:09:39 +0000
Subject: [PATCH] Build Related Changes

---
 config/common.mk | 50 ++++++++++++++++++++++++++----------------------
 1 file changed, 27 insertions(+), 23 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 3e26feb9..edb2a29f 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -3,6 +3,21 @@ $(call inherit-product, vendor/cherish/config/main_version.mk)
 
 TARGET_NO_KERNEL_OVERRIDE := true
 
+PRODUCT_PRODUCT_PROPERTIES += \
+    dalvik.vm.dex2oat64.enabled=true
+
+PRODUCT_SYSTEM_PROPERTIES += \
+    persist.sys.binary_xml=false \
+    ro.cherish.maintainer=ChonDoe
+
+# The Gapps
+$(call inherit-product, vendor/gapps/config.mk)
+
+CHERISH_VANILLA := true
+WITH_THEGAPPS := true
+WITH_ASSISTANT := false
+USE_CCACHE := true
+
 PRODUCT_BRAND ?= CherishOS
 
 PRODUCT_BUILD_PROP_OVERRIDES += BUILD_UTC_DATE=0
@@ -15,11 +30,8 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.com.google.clientidbase=$(PRODUCT_GMS_CLIENTID_BASE)
 endif
 
-# ART
-# Optimize everything for preopt
-PRODUCT_DEX_PREOPT_DEFAULT_COMPILER_FILTER := everything
-# Don't preopt prebuilts
-DONT_DEXPREOPT_PREBUILTS := true
+# Preopt prebuilts
+DONT_DEXPREOPT_PREBUILTS := false
 
 PRODUCT_PROPERTY_OVERRIDES += \
     pm.dexopt.boot=verify \
@@ -42,17 +54,8 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.com.android.dateformat=MM-dd-yyyy \
     persist.sys.disable_rescue=true
 
-ifeq ($(TARGET_BUILD_VARIANT),eng)
-# Disable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-endif
-
 
 # Enable WiFi Display
 PRODUCT_PROPERTY_OVERRIDES += \
@@ -171,7 +174,8 @@ PRODUCT_PACKAGES += \
 
 # Dex preopt
 PRODUCT_DEXPREOPT_SPEED_APPS += \
-    SystemUI
+    SystemUI \
+    Launcher3QuickStep
 
 # SystemUI plugins
 PRODUCT_PACKAGES += \
@@ -205,11 +209,11 @@ PRODUCT_PRODUCT_PROPERTIES += \
     setupwizard.feature.day_night_mode_enabled=true \
     setupwizard.feature.portal_notification=true
 
-TARGET_BUILD_GRAPHENEOS_CAMERA ?= false
-ifeq ($(strip $(TARGET_BUILD_GRAPHENEOS_CAMERA)),true)
-PRODUCT_PACKAGES += \
+#TARGET_BUILD_GRAPHENEOS_CAMERA ?= false
+#ifeq ($(strip $(TARGET_BUILD_GRAPHENEOS_CAMERA)),true)
+#PRODUCT_PACKAGES += \
     GrapheneCamera
-endif
+#endif
 
 # StorageManager configuration
 PRODUCT_PRODUCT_PROPERTIES += \
@@ -245,7 +249,7 @@ PRODUCT_PRODUCT_PROPERTIES += \
 TARGET_SUPPORTS_GOOGLE_RECORDER ?= true
 TARGET_INCLUDE_STOCK_ARCORE ?= false
 TARGET_INCLUDE_LIVE_WALLPAPERS ?= false
-TARGET_SUPPORTS_QUICK_TAP ?= false
+TARGET_SUPPORTS_QUICK_TAP ?= true
 TARGET_USES_MINI_GAPPS ?= false
 
 # Face Unlock
@@ -272,10 +276,10 @@ $(call inherit-product, vendor/cherish/config/bootanimation.mk)
 # Fonts
 $(call inherit-product, vendor/cherish/config/fonts.mk)
 
-ifeq ($(EXTRA_UDFPS_ANIMATIONS),true)
-PRODUCT_PACKAGES += \
+#ifeq ($(EXTRA_UDFPS_ANIMATIONS),true)
+#PRODUCT_PACKAGES += \
     UdfpsResources
-endif
+#endif
 
 # Build
 ifeq ($(CHERISH_VANILLA), true)
-- 
2.17.1

