From a5a348a087b9d06b9e874a54852bc27f02f449dc Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Fri, 10 Jun 2022 14:41:27 +0000
Subject: [PATCH] Env Setup CCACHE

---
 build/envsetup.sh | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index 959a2fb4..45228604 100644
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -676,3 +676,20 @@ export SKIP_ABI_CHECKS=true
 # Override host metadata to make builds more reproducible and avoid leaking info
 export BUILD_USERNAME=nobody
 export BUILD_HOSTNAME=android-build
+
+# check and set ccache path on envsetup
+if [ -z ${CCACHE_EXEC} ]; then
+        ccache_path=$(which ccache)
+        if [ ! -z "$ccache_path" ]; then
+	mkdir /mnt/ccache
+	mount --bind ~/.ccache /mnt/ccache
+	export USE_CCACHE=1
+	export CCACHE_COMPRESS=1
+	export CCACHE_EXEC=/usr/bin/ccache
+	export CCACHE_DIR=/mnt/ccache
+	ccache -M 50G -F 0
+        echo "ccache found and CCACHE_EXEC has been set to : $ccache_path"
+    else
+        echo "ccache not found/installed!"
+    fi
+fi
-- 
2.17.1

