From ba7ee6dc97741b839e7de835469b574a27933764 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Tue, 3 May 2022 20:31:00 +0000
Subject: [PATCH] Add Maintainer & rm adb secure statement

---
 config/common.mk | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index f924c946..797b4639 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -7,7 +7,8 @@ PRODUCT_PRODUCT_PROPERTIES += \
     dalvik.vm.dex2oat64.enabled=true
 
 PRODUCT_SYSTEM_PROPERTIES += \
-    persist.sys.binary_xml=false
+    persist.sys.binary_xml=false \
+    ro.cherish.maintainer=ChonDoe
 
 # The Gapps
 $(call inherit-product, vendor/gapps/config.mk)
@@ -52,17 +53,8 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.com.android.dateformat=MM-dd-yyyy \
     persist.sys.disable_rescue=true
 
-ifeq ($(TARGET_BUILD_VARIANT),eng)
-# Disable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-endif
-
 
 # Enable WiFi Display
 PRODUCT_PROPERTY_OVERRIDES += \
-- 
2.17.1

