From 5c9d486c5b0c1d7a97eb45067cf1bd3ebe7d43bd Mon Sep 17 00:00:00 2001
From: ChonDoit <42556721+ChonDoit@users.noreply.github.com>
Date: Sat, 29 Jan 2022 13:17:49 -0300
Subject: [PATCH] Add Offline Charge Service

From Victor Bo and Additions for Smart Charging
---
 rootdir/init.rc | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 9f38b7d8ce..fd78fce05b 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -11,6 +11,15 @@ import /vendor/etc/init/hw/init.${ro.hardware}.rc
 import /system/etc/init/hw/init.usb.configfs.rc
 import /system/etc/init/hw/init.${ro.zygote}.rc
 
+service charger /bin/charger
+    class charger
+    user system
+    group system shell input graphics wakelock
+    
+on boot
+    chown system system /sys/class/power_supply/batery/input_suspend
+    chmod 0777 /sys/class/power_supply/batery/input_suspend
+
 # Cgroups are mounted right before early-init using list from /etc/cgroups.json
 on early-init
     # Disable sysrq from keyboard
